polylogue/drive_client.py

```
@@ -48,7 +48,8 @@ def ensure_credentials(self) -> Path:
             return env_copy
         if self.ui.plain:
             raise SystemExit(
-                "Missing credentials.json. Set ${DRIVE_CREDENTIAL_ENV} or download a Google OAuth client secret and place it next to polylogue.py."
+                f"Missing credentials.json. Set ${DRIVE_CREDENTIAL_ENV} or download a Google OAuth client secret "
+                "and place it next to polylogue.py."
             )
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)